dvc/serialize.py

```diff
@@ -24,14 +24,17 @@
 def _get_outs(stage: "PipelineStage"):
     outs_bucket = {}
     for o in sort_by_path(stage.outs):
-        bucket_key = ["metrics"] if o.metric else ["outs"]
+        key = "metrics" if o.metric else "outs"
 
-        if not o.metric and o.persist:
-            bucket_key += ["persist"]
+        extra = {}
+        if o.persist:
+            extra["persist"] = True
```

Comment:
Just matching existing style in this method, so using plain strings. Maybe we should discuss strings vs constants one more time, to see how people feel about it these days.